src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -176,6 +188,58 @@ public Claim requestMoreTimeForResponse(String externalId, String authorisation)
         return claim;
     }
 
+    @SuppressWarnings("unchecked")
+    public AboutToStartOrSubmitCallbackResponse requestMoreTimeOnPaper(
```

Comment:
`ClaimService` is getting quite big, maybe it would be worth moving this to another class? Also this is CCD related.

Reply:
270 lines, not sure if its worth it.